README.md

```diff
@@ -29,3 +29,20 @@ long as we can clearly make them out, we say “they're there”.
 3. Train MobileNet.
 4. See how well MobileNet does.
 5. Try deploying MobileNet to David's Raspberry Pi 3.
+
+### TODO Mobilenet:
```

Comment:
Please leave *two* lines blank before and after headings in markdown and don't end headings in colons.